src/ui.message.js

```diff
@@ -5,6 +5,7 @@ var _ = Mavo.UI.Message = $.Class({
 		this.mavo = mavo;
 		this.message = message;
 		this.closed = Mavo.promise();
+		this.o = o;
```

Comment:
`o` is convenient for a local variable, but means nothing as a property. The conventional name for this is `this.options`.

Reply:
I switched that. But when making the change I noticed that there had been a previous bug in the code so I fixed that too. 

`this.options.style.transition === "none" ? Promise.resolve() : $.transition(this.element, {opacity: 0});` throws an error when this.options.style is undefined so I had to rewrite it to

```
if (this.options.style) {
	var promise = this.options.style.transition === "none" ? Promise.resolve() : $.transition(this.element, {opacity: 0});
} 
else {
	var promise = Promise.resolve();
}
``` 

I am pushing the changes now

Reply:
You can use `$.value(this.options, "style", "transition") === "none"` to avoid such errors.

Reply:
I just rewrote it to `var closed = this.options.style && this.options.style.transition !== "none" ?  $.transition(this.element, {opacity: 0}) : Promise.resolve();`. Is that fine?